assets/queries/terraform/aws/api_gateway_with_open_access/query.rego

```diff
@@ -0,0 +1,17 @@
+package Cx
+
+CxPolicy[result] {
+	document := input.document[i]
+	resource = document.resource.aws_api_gateway_method[name]
+
+	resource.authorization == "NONE"
+	resource.http_method != "OPTIONS"
+
+	result := {
+		"documentId": document.id,
+		"searchKey": sprintf("aws_api_gateway_method[%s].authorization", [name]),
```

Comment:
Should be "http_method" instead of "authorization"?

"searchKey": sprintf("aws_api_gateway_method[%s].http_method", [name]),